Leanplum-SDK/Classes/LPNetworkOperation.m

```diff
@@ -175,6 +171,10 @@ - (void)runSynchronously:(BOOL)synchronous
                 }
```

Comment:
Yes this will trigger it. Will it have consequences? 
This was an intentional change to make sure we return after executing completely but sendNow handles this: 
https://github.com/Leanplum/Leanplum-iOS-SDK/blob/master/Leanplum-SDK/Classes/LeanplumRequest.m#L407

I will revert it.